text2sql-backend/dataline/models/base.py

```diff
@@ -18,6 +19,5 @@ class DBModel(MappedAsDataclass, DeclarativeBase, init=False, kw_only=True):
     )
 
 
-class UUIDMixin:
-    # Generate UUID in python
-    id: Mapped[UUID] = mapped_column("id", DB_UUID, primary_key=True, default=uuid.uuid4, init=False)
+class UUIDMixin(MappedAsDataclass, init=False, kw_only=True):
+    id: Mapped[UUID] = mapped_column("id", DB_UUID, primary_key=True, insert_default=uuid.uuid4, init=False)
```

Comment:
What does `insert_default` do?

Reply:
`default` is deprecated because it is confusing: it can mean one of two things so they will remove it. The two things are:

`insert_default` means create this upon insertion using this factory function
`default_factory` means this is created when you instantiate a model (not inserted yet, this is at the dataclass level)

We want insert_default here!